EpiAware/src/EpiAwareUtils/EpiAwareUtils.jl

```diff
@@ -37,5 +37,6 @@ include("get_param_array.jl")
 include("SafeInt.jl")
 include("SafePoisson.jl")
 include("SafeNegativeBinomial.jl")
+include("_expand_dist.jl")
```

Comment:
if we are using this across modules I think we need to export it or at least update the name to a non-internal pattern

Reply:
The move to `EpiAwareUtils` was motivated by an idea which I didn't go forwards with so I've reverted.